runner/src/provisioner/schemas/logs-table.ts

```diff
@@ -0,0 +1,61 @@
+export const logsTableDDL = (source: string, schemaName: string) => `
+CREATE TABLE __logs (
+    id BIGSERIAL NOT NULL,
+    block_height NUMERIC(20) NOT NULL,
+    log_date DATE NOT NULL,
+    log_timestamp TIMESTAMP NOT NULL,
+    log_type TEXT NOT NULL,
+    log_level TEXT NOT NULL,
+    message TEXT NOT NULL,
+    PRIMARY KEY (log_date, id)
+) PARTITION BY RANGE (log_date);
+
+CREATE INDEX logs_log_timestamp_idx ON __logs USING btree (log_timestamp);
```

Comment:
```suggestion
CREATE INDEX __logs_log_timestamp_idx ON __logs USING btree (log_timestamp);
```
While conflicts are unlikely, I would still prefix this just to signify it's internal